src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -113,32 +113,26 @@ public function manageAction()
         $filters->setType(AddonListFilterType::MODULE | AddonListFilterType::SERVICE)
             ->removeStatus(AddonListFilterStatus::UNINSTALLED);
         $installedProducts = $moduleRepository->getFilteredList($filters);
-
-        $modules = new stdClass();
-        foreach (['native_modules', 'theme_bundle', 'modules'] as $subpart) {
-            $modules->{$subpart} = [];
-        }
-
-        foreach ($installedProducts as $installedProduct) {
-            $modules->{$this->findModuleType($installedProduct, $modulesTheme)}[] = $installedProduct;
-        }
-
-        foreach ($modules as $moduleLabel => $modulesPart) {
-            $collection = AddonsCollection::createFrom($modulesPart);
-            $modules->{$moduleLabel} = $modulesProvider->generateAddonsUrls($collection);
-            $modules->{$moduleLabel} = $this->get('prestashop.adapter.presenter.module')
-                                           ->presentCollection($modulesPart);
-        }
+        $categories = $this->getCategories($modulesProvider, $installedProducts);
+
+        $bulkActions = [
+            'bulk-uninstall' => $this->trans('Uninstall', 'Admin.Actions'),
+            'bulk-disable' => $this->trans('Disable', 'Admin.Actions'),
+            'bulk-enable' => $this->trans('Enable', 'Admin.Actions'),
+            'bulk-reset' => $this->trans('Reset', 'Admin.Actions'),
+            'bulk-enable-mobile' => $this->trans('Enable Mobile', 'Admin.Modules.Feature'),
+            'bulk-disable-mobile' => $this->trans('Disable Mobile', 'Admin.Modules.Feature'),
+        ];
 
         return $this->render(
             'PrestaShopBundle:Admin/Module:manage.html.twig',
             [
+                'maxModulesDisplayed' => self::MAX_MODULES_DISPLAYED,
+                'bulkActions' => $bulkActions,
                 'layoutHeaderToolbarBtn' => $this->getToolbarButtons(),
-                'layoutTitle' => $this->trans('Manage installed modules', 'Admin.Modules.Feature'),
-                'modules' => $modules,
-                'topMenuData' => $this->getTopMenuData(
-                    $this->get('prestashop.categories_provider')->getCategoriesMenu($installedProducts)
-                ),
+                'layoutTitle' => $this->trans('Module manager', 'Admin.Modules.Feature'),
```

Comment:
Wouldn't it be better to write _Modules manager_ in the plural, I assume that there are several modules managed here, right?

Reply:
It was a Lou request :)

Reply:
Yes, no S in English